torch_geometric/nn/aggr/base.py

```diff
@@ -71,8 +96,15 @@ def reduce(self, x: Tensor, index: Optional[Tensor] = None,
         assert index is not None
         return scatter(x, index, dim=dim, dim_size=dim_size, reduce=reduce)
 
-    def __repr__(self) -> str:
-        return f'{self.__class__.__name__}()'
+    def to_dense_batch(self, x: Tensor, index: Optional[Tensor] = None,
+                       ptr: Optional[Tensor] = None,
+                       dim_size: Optional[int] = None,
+                       dim: int = -2) -> Tuple[Tensor, Tensor]:
+
+        self.assert_index_present(index)  # TODO
```

Comment:
```suggestion
        self.assert_index_present(index)  # TODO: support non-sorted index
```
Unclear what the TODO was for me exactly (not sure I'm right here, suggest a better message if mine is not accurate)

Reply:
done.